tests/icon-api.js

```diff
@@ -560,14 +561,18 @@ test('getBestVariant() - params prioritization', (t) => {
     'same result regardless of variants order'
   )
 
-  t.alike(result, wantedSizeSvgVariant, 'mime type > size > pixel density')
+  assert.deepEqual(
+    result,
+    wantedSizeSvgVariant,
+    'mime type > size > pixel density'
+  )
 })
 
 // TODO: Currently fails. Not sure if we'd run into this situation often in reality
 test(
   'getBestVariant - multiple exact matches return deterministic result',
-  { todo: true },
-  (t) => {
+  { skip: true },
```

Comment:
Changed `todo` to `skip` so errors didn't show up in the output, but this has no other effect (e.g., test will still be skipped either way).